### PR TITLE
Branch attendance feature

### PR DESCRIPTION
**Pull request by kev-intq**
Comments field has been added, but no support for adding/removing comments yet.

TODO: Comments along with participation.

### CODE REVIEW

File: src/main/java/seedu/address/model/attendance/AttendanceList.java [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+            for (Attendance attendance : attendances) {
+                if (attendance.getStudentId().equals(studentId)) {
+                    hasAttendencePresent = true;
+                    break;
+                }
+            }
+
+            if (!hasAttendencePresent) {
+                System.out.println("creating");
+                System.out.println(weeks);
```
Comment (seanlaiys) [inline src/main/java/seedu/address/model/attendance/AttendanceList.java:63]: For now will you still need this line to check whether you method works?

File: src/main/java/seedu/address/model/attendance/AttendanceList.java [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+    }
+
+    /**
+     * Marks the attendance for the specified student.
+     *
+     * @param studentId the NusNetId of a student.
+     * @param week the week that the attendance should be marked for the student.
+     */
+    public void markAttendanceForStudent(NusNetId studentId, int week) {
+        System.out.println(studentId);
```
Comment (seanlaiys) [inline src/main/java/seedu/address/model/attendance/AttendanceList.java:97]: Similar to above, will we still need this?

**Review by seanlaiys — APPROVED** [review]
Ok to merge! Just need to double check if we still need the output in the terminal to validate if the methods work.

File: src/main/java/seedu/address/model/attendance/AttendanceList.java (hunk 2 above)
Comment (kev-intq) [reply src/main/java/seedu/address/model/attendance/AttendanceList.java:97]: Thanks for pointing it out. I'll remove it!

File: src/main/java/seedu/address/model/attendance/AttendanceList.java (hunk 1 above)
Comment (kev-intq) [reply src/main/java/seedu/address/model/attendance/AttendanceList.java:63]: Thanks for pointing it out. I'll remove it!

**Pull request merged**